.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+.env
```

Comment:
Remind me -- what is the new environment variable that is necessary for this feature and what is it used for?